acceptance/tests/windows/stop_puppet_svc_stops_puppet.rb

```diff
@@ -0,0 +1,48 @@
+test_name '(C14941) Stopping puppet service on Windows stops puppet' do
+
+  confine :to, :platform => 'windows'
+  
+  opts = {
+    :acceptable_exit_codes => [0],
+  }
+  
+  agents.each do
+    step 'Verify the puppet service is running' do
+      opts[:acceptable_exit_codes] << 32
```

Comment:
i had to think about this for a second.  could probably use a comment to say we WANT to exit successfully on either 0 or 32 to confirm `RUNNING` later